src/data/markdown/translated-guides/en/02 Using k6/04 Thresholds.md

```diff
@@ -173,30 +173,23 @@ You can find more specific threshold examples on the [Counter](/javascript-api/k
 
 ## Threshold Syntax
 
-Thresholds can be specified in a short or full format.
+A threshold requires at least one `threshold_expression` to be defined:
 
 <CodeGroup labels={["threshold-options.js"]} lineNumbers={[true]}>
 
 ```javascript
 export const options = {
   thresholds: {
-    metric_name1: ['threshold_expression' /* ...*/], // short format
-    metric_name1: [
-      {
-        threshold: 'threshold_expression',
-        abortOnFail: true, // boolean
-        delayAbortEval: '10s', // string
-      },
-    ], // full format
-  },
+    metric_name: [ 'threshold_expression', ... ], // short format
+  }
```

Comment:
Hmm, I'd leave the example with the full format here, just fix it so it has `metric_name1` and `metric_name2`. It doesn't seem like we explain it anywhere else in the page...

Reply:
Yes, I wondered about this part too.. But the full format is explained in one of the next sections: https://k6.io/docs/using-k6/thresholds/#aborting-a-test-when-a-threshold-is-crossed
I'm fine putting it back with another name, as you suggested too, I just find it a bit strange that description is in another place.

Reply:
Doesn't hurt to have the example with both formats on the top :man_shrugging: